src/sotoki/scraper.py

```diff
@@ -77,9 +77,15 @@ def sanitize_inputs(self):
             self.conf.title = Global.site["LongName"]
         self.conf.title = self.conf.title.strip()
 
-        if not self.conf.description:
-            self.conf.description = Global.site["Tagline"]
-        self.conf.description = self.conf.description.strip()
+        default_description = Global.site["Tagline"].strip()
+        if self.conf.description is not None:
+            user_description = self.conf.description.strip()
+        else:
+            user_description = None
+        user_long_description = self.conf.long_description
+        description, long_description = compute_descriptions(default_description, user_description, user_long_description)
```

Comment:
You can rewrite this as a one-liner: `self.conf.description, self.conf.long_description = compute_descriptions(...)`